### PR TITLE
corrected placement of options

### PR DESCRIPTION
**Pull request by KyleDanny**

### CODE REVIEW

**Review by ssb-cgn — APPROVED** [review]

File: src/main/resources/assets/js/app/highchart.es6 [hunk 1]
```diff
@@ -43,85 +43,93 @@ const createSetOptions = {
...
+      drillUpText: 'Tilbake til {series.name}',
+      loading: 'Tegner graf...',
+      noData: 'Tall ikke tilgjengelig',
+      numericSymbols: [null, ' mill.', ' mrd.'],
+      printChart: 'Skriv ut graf',
+      resetZoom: 'Nullstill zoom',
+      resetZoomTitle: 'Nullstill zoom',
+      thousandsSep: ' ',
     },
-    svgContainerLabel: 'Interaktiv graf',
```
Comment (ssb-cgn) [inline src/main/resources/assets/js/app/highchart.es6:107]: Dont move this, only chartTypes and Series

**Review by ssb-cgn — CHANGES_REQUESTED** [review]
Fix phrases download and others

File: src/main/resources/assets/js/app/highchart.es6 (hunk 1 above)
Comment (KyleDanny) [reply src/main/resources/assets/js/app/highchart.es6:107]: Har flyttet tilbake :) men jeg tror disse 3 må ligge under 'accessibility': 

```
credits: 'Diagramkreditter: {creditsStr}.',
svgContainerLabel: 'Interaktiv graf',
defaultChartTitle: 'Graf',
```

Comment (KyleDanny) [reply src/main/resources/assets/js/app/highchart.es6:107]: https://api.highcharts.com/highcharts/lang.accessibility

**Review by ssb-cgn — APPROVED** [review]
Very good 👍

**Pull request merged**